doc/common_pitfalls.rst

```diff
@@ -416,7 +416,7 @@ it will allow the estimator RNG to vary for each fold.
 **Cloning**
 
 Another subtle side effect of passing `RandomState` instances is how
-:func:`~sklearn.clone` will work::
+:func:`~sklearn.base.clone` will work::
```

Comment:
But it is also available at `sklearn.clone`. Can we appease sphinx some other way than having to tell people that they need to go via `sklearn.base` when it isn't true?

Reply:
note that this is rendered as `clone`, it's not showing users how to import it really. I'm happy either way.

Reply:
As mentioned by @adrinjalali, the rendering does not change due to the tild `~`.
Here, we don't have the case, but I would agree with @ogrisel to advocate for importing from `sklearn.base` since this is already what we have in the `API` documentation. However, I don't see anywhere in the documentation such occurrences anyway.